src/main/webapp/app/modules/provider/record/single-record-view.tsx

```diff
@@ -0,0 +1,287 @@
+import './single-record-view.scss';
+
+import React from 'react';
+import { Collapse, Button, CardBody, Card, CardTitle } from 'reactstrap';
+import { Translate } from 'react-jhipster';
+import { Link, RouteComponentProps } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { connect } from 'react-redux';
+import { IRootState } from 'app/shared/reducers';
+import { getProviderEntity } from 'app/entities/organization/organization.reducer';
+// @ts-ignore
+import BuildingLogo from '../../../../static/images/building.svg';
+// @ts-ignore
+import PeopleLogo from '../../../../static/images/people.svg';
+// @ts-ignore
+import ServiceLogo from '../../../../static/images/service.svg';
+
+export interface ISingleRecordViewProps extends StateProps, DispatchProps, RouteComponentProps<{ orgId: string }> {
+  record: any;
+}
+
+export interface ISingleRecordViewState {
+  serviceWidths: any[];
+  locationWidths: any[];
+  isOrganizationOpen: boolean;
+  isServicesOpen: boolean;
+  isLocationsOpen: boolean;
+}
+
+class SingleRecordView extends React.Component<ISingleRecordViewProps, ISingleRecordViewState> {
+  state: ISingleRecordViewState = {
+    serviceWidths: [],
+    locationWidths: [],
+    isOrganizationOpen: false,
+    isServicesOpen: false,
+    isLocationsOpen: false
+  };
+
+  toggleOrganization = () => this.setState({ isOrganizationOpen: !this.state.isOrganizationOpen });
+  toggleServices = () => this.setState({ isServicesOpen: !this.state.isServicesOpen });
+  toggleLocations = () => this.setState({ isLocationsOpen: !this.state.isLocationsOpen });
+
+  componentDidMount() {
+    this.props.getProviderEntity(this.props.match.params.orgId);
+  }
+
+  render() {
+    const { isOrganizationOpen, isServicesOpen, isLocationsOpen } = this.state;
+    const { organization } = this.props;
+    const locationsCount = organization && organization.locations ? organization.locations.length : 0;
+    const servicesCount = organization && organization.services ? organization.services.length : 0;
+
+    return (
+      <div className="background single-record-view">
+        <Button tag={Link} to="/" color="" className="d-none d-sm-block position-fixed">
```

Comment:
Isn't d-none cause to hide component?

Reply:
There is d-none d-sm-block which means, that on a small resolution this will be hidden and on a bigger, it will be shown.